lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -38,9 +40,13 @@ class AwsCompileScheduledEvents {
               }
               ScheduleExpression = event.schedule.rate;
               State = event.schedule.enabled ? 'ENABLED' : 'DISABLED';
+              Input = event.schedule.input || '';
```

Comment:
I had also had to escape the `Input` variable so the stringified Object string won't break `JSON.parse` below.